packages/flutter/test/material/divider_test.dart

```diff
@@ -22,6 +22,61 @@ void main() {
     expect(find.byType(Divider), paints..path(strokeWidth: 0.0));
   });
 
+  testWidgets('Horizontal divider custom indentation', (WidgetTester tester) async {
+    Rect dividerRect;
+    Rect lineRect;
+
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: Center(
+          child: Divider(
+            indent: 10.0,
+          ),
+        ),
+      ),
+    );
+
+    // The divider line is drawn with a DecoratedBox with a border
+    dividerRect = tester.getRect(find.byType(Divider));
+    lineRect = tester.getRect(find.byType(DecoratedBox));
+    expect(lineRect.left, dividerRect.left + 10.0);
+    expect(lineRect.right, dividerRect.right);
+
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: Center(
+          child: Divider(
+            endIndent: 10.0,
+          ),
+        ),
+      ),
+    );
+
+    dividerRect = tester.getRect(find.byType(Divider));
+    lineRect = tester.getRect(find.byType(DecoratedBox));
+    expect(lineRect.left, dividerRect.left);
+    expect(lineRect.right, dividerRect.right - 10.0);
+
+    await tester.pumpWidget(
+      const Directionality(
+        textDirection: TextDirection.ltr,
+        child: Center(
+          child: Divider(
+            indent: 10.0,
+            endIndent: 10.0,
+          ),
+        ),
+      ),
+    );
+
+    dividerRect = tester.getRect(find.byType(Divider));
+    lineRect = tester.getRect(find.byType(DecoratedBox));
+    expect(lineRect.left, dividerRect.left + 10.0);
+    expect(lineRect.right, dividerRect.right - 10.0);
```

Comment:
Consider making the `10.0` a constant since it's used so much?  Up to you, I think it's clear either way.